src/io/stakater/vc/Git.groovy

```diff
@@ -140,29 +139,11 @@ def generateVersionAndPush(def versionFile, def repoName, def repoOwner){
   return version
 }
 
-def generateVersionAndPush(def versionFile, def repoName, def repoOwner, def goProjectPath){
-  echo "Generating New Version"
-  def common = new io.stakater.Common()
-  def version = common.shOutput("jx-release-version --gh-owner=${repoOwner} --gh-repository=${repoName} --version-file ${versionFile}")
-  sh """
-      echo -n "${version}" > ${versionFile}
-  """
-  commitChanges(goProjectPath, "Bump Version to ${version}")
-  echo "Pushing Tag ${version} to Git"
-  createTagAndPush(goProjectPath, version)
-  return version
-}
-
-def tagAndRelease(def versionFile, def repoName, def repoOwner){
-  def version = generateVersionAndPush(versionFile, repoName, repoOwner)
+def tagAndRelease(def versionFile, def version, def repoName, def repoOwner){
+  def version = generateVersionAndPush(versionFile, version, repoName, repoOwner)
   createRelease(version)
 }
 
-def createReleaseViaGoReleaser(def versionFile, def repoName, def repoOwner){
```

Comment:
See it's usage [here](https://github.com/stakater/stk/blob/master/Jenkinsfile#L90)